app/build.gradle

```diff
@@ -40,12 +40,18 @@ android {
             signingConfig signingConfigs.config
         }*/
     }
-
+    
+    android.buildTypes.each { type ->
+        type.buildConfigField 'String', 'RC_MAPS_API_KEY', RC_MAPS_API_KEY
+    }
 
     repositories {
         maven {
             url "https://raw.github.com/dimelo/Dimelo-Android/master"
```

Comment:
while we're at it can you change this link for `https://raw.github.com/ringcentral/engage-digital-messaging-android` and make sure it still works fine?